src/components/BlogPost/BlogPost.js

```diff
@@ -46,24 +100,51 @@ function BlogPost(props) {
           <Grid>{blogSideDate}</Grid>
           {blogSideTags}
         </Grid>
-        <Grid item xs={9}>
+        <Grid item lg={9} xs={12}>
           <Link
+            className={width === 'xs' ? classes.linkSmall : classes.link}
             to={{
               pathname: `/blogs/${id}`,
             }}
           >
-            <h1>{name}</h1>
+            <span>{name}</span>
           </Link>
+          {/* <div
```

Comment:
Do you need this code still? If not, delete it along with the associated styling. Cleaner the code, the easier it is to debug or refactor in the future.